addons/payment_mercado_pago/const.py

```diff
@@ -27,6 +27,17 @@
     'VES',  # Sovereign Bolivar
 ]
 
+# The codes of the payment methods to activate when Mercado Pago is activated.
+DEFAULT_PAYMENT_METHODS_CODES = [
+    # Primary payment methods.
+    'card',
+    # Brand payment methods.
+    'visa',
+    'mastercard',
+    'amex',
+    'discover',
+]
```

Comment:
Could you check if there isn't a card brand typically added by Mercado Pago? If so, it'd make sense to enable it.